app/models/invenio_rdm_record_converter.rb

```diff
@@ -130,7 +130,7 @@ def invenio_metadata(gf)
       "description": gf.description.first,
       "additional_descriptions": additional_descriptions(gf.description),
       "publisher": gf.publisher.shift,
-      "publication_date": "#{gf.date_uploaded.year}-#{gf.date_uploaded.month}-#{gf.date_uploaded.day}",
```

Comment:
I'm not sure why I did this in the first place. I'm going to run a test export to make sure that this wasn't intentional.